### PR TITLE
Fixed a typo

### PR DESCRIPTION
**Pull request by Soupstraw**
Fixed an unmatched open parentheses

### CODE REVIEW

File: eras/alonzo/formal-spec/utxo.tex [hunk 1]
```diff
@@ -18,7 +18,7 @@ \section{UTxO}
     & \fun{feesOK} : \PParams \to \Tx \to \UTxO \to \Bool  \\
     & \fun{feesOK}~\var{pp}~tx~utxo~= \\
     &~~      \minfee{pp}~{tx} \leq \txfee{txb} \wedge (\fun{txrdmrs}~tx \neq \emptyset \Rightarrow \\
-    &~~~~~~((\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
+    &~~~~~~(\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
     &~~~~~~\wedge~ \fun{isAdaOnly}~\var{balance} \\
     &~~~~~~\wedge~ \var{balance}*100 \geq \txfee{txb} * (\fun{collateralPercent}~pp) \\
     &~~~~~~\wedge~ \fun{collateral}~{tx} \neq \emptyset) \\
```
Comment (JaredCorduan) [inline eras/alonzo/formal-spec/utxo.tex:24]:
```suggestion
    &~~~~~~\wedge~ \fun{collateral}~{tx} \neq \emptyset)) \\
```

File: eras/alonzo/formal-spec/utxo.tex [hunk 2]
```diff
@@ -18,7 +18,7 @@ \section{UTxO}
     & \fun{feesOK} : \PParams \to \Tx \to \UTxO \to \Bool  \\
     & \fun{feesOK}~\var{pp}~tx~utxo~= \\
     &~~      \minfee{pp}~{tx} \leq \txfee{txb} \wedge (\fun{txrdmrs}~tx \neq \emptyset \Rightarrow \\
-    &~~~~~~((\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
+    &~~~~~~(\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
```
Comment (JaredCorduan) [inline eras/alonzo/formal-spec/utxo.tex:21]:
```suggestion
    &~~~~~~((\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
```

Comment (JaredCorduan) [reply eras/alonzo/formal-spec/utxo.tex:21]: I think it might be clearer to instead keep those parens, but close the outermost one. yea?

Comment (Soupstraw) [reply eras/alonzo/formal-spec/utxo.tex:21]: I think the line breaks already help break up the conjunctions visually, so I'd prefer if we removed the parentheses

Comment (JaredCorduan) [reply eras/alonzo/formal-spec/utxo.tex:21]: hmm. but the line break isn't really semantically meaningful, I feel like the parens are much more explicit. let's see what @WhatisRT thinks.

Comment (Soupstraw) [reply eras/alonzo/formal-spec/utxo.tex:21]: conjunction binds more strongly than implication so the parentheses wouldn't change anything

Comment (WhatisRT) [reply eras/alonzo/formal-spec/utxo.tex:21]: I'd happily remove even the pair of parentheses around the forall line as well ;)

Comment (JaredCorduan) [reply eras/alonzo/formal-spec/utxo.tex:21]: ah, I was worried about the scope of the quantifier, but that's still covered with this deletion. great!

**Review by JaredCorduan — APPROVED** [review]
great catch!

**Pull request merged**